utils/helper_cloud.sh

```diff
@@ -1,17 +1,37 @@
 #!/bin/bash
 
 ################################################################
-# Source Confluent Platform versions
+# helper_cloud.sh
+# --------------------------------------------------------------
+# This library of functions automates common tasks with Confluent Cloud https://confluent.cloud/ 
+# These are intended to be examples for demos and testing.
+#
+# Example usage in https://github.com/confluentinc/examples
+#
+# Get the library:
+#
+#   wget -O helper_cloud.sh https://raw.githubusercontent.com/confluentinc/examples/latest/utils/helper_cloud.sh
+#   source ./helper_cloud.sh
+#
```

Comment:
These instructions do not work as the config.env file isn't brought over, here is the error I receive:

```
08:15:01 in dev/tmp/helper_cloud
[I] ➜ wget -O helper_cloud.sh https://raw.githubusercontent.com/confluentinc/examples/latest/utils/helper_cloud.sh
--2020-06-01 08:15:02--  https://raw.githubusercontent.com/confluentinc/examples/latest/utils/helper_cloud.sh
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 199.232.76.133
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|199.232.76.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 27001 (26K) [text/plain]
Saving to: ‘helper_cloud.sh’

helper_cloud.sh                               100%[=================================================================================================>]  26.37K  --.-KB/s    in 0.02s

2020-06-01 08:15:02 (1.46 MB/s) - ‘helper_cloud.sh’ saved [27001/27001]


08:15:02 in dev/tmp/helper_cloud
[I] ➜ source ./helper_cloud.sh
./helper_cloud.sh:.:7: no such file or directory: /Users/rspurgeon/dev/tmp/helper_cloud/config.env
```

Reply:
@rspurgeon that's because you are pulling on `latest` branch, not this branch which removes the dependency on `config.env`

Reply:
Thanks @ybyzek, I see now. I just copy and pasted from the PR for testing, bad assumption.

On my second attempt I get the following issue:

```
09:41:13 in dev/tmp/helper_cloud
[I] ➜ wget -O helper_cloud.sh https://raw.githubusercontent.com/confluentinc/examples/DEVX-1824/utils/helper_cloud.sh
--2020-06-01 09:41:17--  https://raw.githubusercontent.com/confluentinc/examples/DEVX-1824/utils/helper_cloud.sh
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 199.232.76.133
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|199.232.76.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 30281 (30K) [text/plain]
Saving to: ‘helper_cloud.sh’

helper_cloud.sh                               100%[=================================================================================================>]  29.57K  --.-KB/s    in 0.02s

2020-06-01 09:41:18 (1.54 MB/s) - ‘helper_cloud.sh’ saved [30281/30281]


09:41:18 in dev/tmp/helper_cloud
[I] ➜ source ./helper_cloud.sh
./helper_cloud.sh:466: unmatched `
```

Reply:
Ah, I just snuck that change in w/out testing.  Good catch, now fixed in last commit, thanks.